statsmodels/discrete/discrete_model.py

```diff
@@ -406,12 +406,29 @@ def predict(self, params, exog=None, linear=False):
         raise NotImplementedError
 
     def _derivative_exog(self, params, exog=None, dummy_idx=None,
-            count_idx=None):
+                         count_idx=None):
         """
         This should implement the derivative of the non-linear function
         """
         raise NotImplementedError
 
+    def _wrap_derivative_exog(self, margeff, params, exog, dummy_idx,
```

Comment:
"wrap" is not a good a good name because we use "wrap"ping mainly for pandas and output wrapping

Reply:
I'm open to suggestions

Reply:
_derivative_exog_helper

Reply:
_derivative_exog_formatter

Reply:
_derivative_exog_packager